src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    return function(b) {
+      counter++;
+
+      if (counter <= 3) {
+        return a + b;
```

Comment:
you can combine both statements that return a + b, a then return an error in any other case